core/src/scene/sceneLoader.cpp

```diff
@@ -101,7 +101,15 @@ bool SceneLoader::loadScene(Node& config, Scene& _scene) {
     std::sort(_scene.styles().begin(), _scene.styles().end(),
               [](std::unique_ptr<Style>& a, std::unique_ptr<Style>& b) {
                   if (a->blendMode() != b->blendMode()) {
-                      return a->blendMode() == Blending::none;
+                      if (a->blendMode() == Blending::none || b->blendMode() == Blending::none) {
+                          return a->blendMode() == Blending::none;
```

Comment:
With respect to tie breaking, that is already happening for us (https://github.com/tangrams/tangram-es/blob/blend-order/core/src/scene/sceneLoader.cpp#L116).

Yes, I agree on the other 2 suggestions. However, will have to use c style enum, instead of c++11 strongly typed enum here, for `<<` operator to work.